http/http_stream.hpp

```diff
@@ -99,6 +109,7 @@ class ConnectionImpl : public Connection
                 }
             });
     }
+
```

Comment:
Please try to avoid white space changes for next time

Reply:
sure